bots/tests-invoke

```diff
@@ -49,6 +49,7 @@ def main():
             action='store', help='Publish results centrally to a sink')
     parser.add_argument('-v', '--verbose', action='store_true', help='Verbose output')
     parser.add_argument('context', help="The context or type of integration tests to run")
+    parser.add_argument('pull_number', default=None, help="The number of the pull request to test")
```

Comment:
I have sort of a déjà-vu here, but I think this should be an actual option `--pull-number` for clarity. Optional positional arguments are really confusing. Then you can also drop the `default=`.

To clarify, this is only being used for collision detection, right?